Sources/UtilityBeltNetworking/Models/MimeType.swift

```diff
@@ -0,0 +1,5 @@
+// Copyright © 2021 SpotHero, Inc. All rights reserved.
+
+public enum MimeType: String, Codable {
+    case json = "application/json"
+}
```

Comment:
I'm still thinking this should be a struct so that `ensureMimeType` can be expanded upon without necessarily requiring updates within the `UtilityBelt` layer.

For example, I use UB for a lot of non-SH projects as a way of dogfooding apps with this library, and there are one or two cases I've needed custom mime types that a library might not (and shouldn't) account for by default. This is similar to why the Headers dictionary is keyed with `Strings` under the hood--there are custom headers for various use cases and we need to allow someone to override it.

Basically my own rule of thumb is: if something is _not_ a definitive, finite list (not implying it can't grow, but that we know all the possible values _at this time_ and don't see an immediate need for other values), it needs to _not_ be an `enum` type or we back ourselves into a corner. Does that make sense?